### PR TITLE
Add validator monitor

### PR DESCRIPTION
**Pull request by kaymakf**

### CODE REVIEW

File: projects/deus-validator-monitor/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Validator Monitor",
+    "type": "Tools",
+    "image": "deus.png",
+    "state": 1,
+    "twitter_link": "twitter.com/deuslabs",
+    "tg_link": "",
+    "address": "",
+    "github_link": "",
```
Comment (orkunkl) [inline projects/deus-validator-monitor/index.json]: why not fill in these too?

File: projects/deus-validator-monitor/index.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Validator Monitor",
+    "type": "Tools",
+    "image": "deus.png",
+    "state": 1,
+    "twitter_link": "twitter.com/deuslabs",
+    "tg_link": "",
```
Comment (kaymakf) [inline projects/deus-validator-monitor/index.json]:
```suggestion
    "tg_link": "https://t.me/deuslabs",
```

File: projects/deus-validator-monitor/index.json (hunk 1 above)
Comment (kaymakf) [reply projects/deus-validator-monitor/index.json]:
```suggestion
    "github_link": "https://github.com/deus-labs/node-tooling/tree/main/grafana-dashboards",
```

File: projects/deus-validator-monitor/index.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+    "name": "Validator Monitor",
+    "type": "Tools",
+    "image": "deus.png",
+    "state": 1,
+    "twitter_link": "twitter.com/deuslabs",
```
Comment (kaymakf) [inline projects/deus-validator-monitor/index.json]:
```suggestion
    "twitter_link": "https://twitter.com/deuslabs",
```

**Review by orkunkl — APPROVED** [review]

**Review by blockcreators — APPROVED** [review]

**Pull request merged**